modules/administration-guide/partials/con_che-server-with-devworkspace.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// {prod-id-short}-workspace-controller-with-devworkspace
+
+
+[id="{prod-id-short}-server-with-devworkspace_{context}"]
+= {prod-short} server
+
+The {prod-short} server is the central service of {prod-short} server-side components. 
+It simplifies the usage of the {devworkspace} API in a multitenant context.
+It is a Java web service exposing an HTTP REST API. 
+The {prod-short} deployment starts it in the `{prod-deployment}` Deployment. 
+
+It need access to:
```

Comment:
To avoid ambiguity, I would say, `Che server needs access to:` 

This way, we don't doubt that is is Che serve that needs access to these registries and engines.